bears/python/PyUnusedCodeBear.py

```diff
@@ -15,16 +15,16 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
 
         - Unneeded pass statements.
-        - Unneeded builtin imports.
+        - All unneeded imports.
 
         :param remove_all_unused_imports:
-            True removes all unused imports - might have side effects
+            False removes only unused builtin imports - might have side effects
```

Comment:
According to https://github.com/coala/coala-bears/pull/1844#discussion_r122526269 and https://github.com/coala/coala-bears/pull/1829#discussion_r122533138 , better move `- might have side effects` to the changed list entry above